server/gendb/gendb.go

```diff
@@ -137,8 +139,21 @@ func findCompositeAttribute(config XDBConfig, attributeName string) *CompositeAt
 
 }
 
+func extractModuleNameAndVersionSuffix(packageName, outputPath string) (moduleName, versionSuffix string) {
+	vregex, err := regexp.Compile("/v[0-9]$|/v[0-9][0-9]")
```

Comment:
Any reason to only support two-digit major versions?

Reply:
That was a compromise that I asked you about pretty early on, it's possible to write this to support larger version numbers but it adds complexity and reduces readability in this line. As of now I believe we have exactly one wag service that has hit 2 digit version numbers. I don't expect us to get to anything have a three digit version number.